MenMa/Managers/NetworkManager.swift

```diff
@@ -47,9 +47,11 @@ extension Venue {
                         guard let name = venue["name"] as? String else { return nil }
                         guard let id = venue["id"] as? String else { return nil}
                         let url = venue["url"] as? String
+                        let location = venue["location"] as? Dictionary<String, AnyObject>
+                        let distance = location!["distance"] as? Int
```

Comment:
Generally speaking, you should avoid using forced unwrapping (!) as much as possible. It causes the app to get crashed if you misuse it.

Subscription of Dictionary in Swift returns value of type optional, because a Dictionary's key might not exist. So, location is of type Dictionary<String, AnyOjbect> of optional here. To use its value, location needs to be unwrapped.

There are several ways unwrapping optional: 1) if let optional binding. 2) guard 3) ?? nil coalescing operator 4) and maybe something else.

At line 50, if you use guard, you could remove forced unwrapping at line 51.

My idea was to have lat property, lng property, and distance property, or location property of type CLLocation. The cons of CLLocation property is that you can just pass it to anywhere CLLocation is an argument, e.g., CLLocation could be used for the watch map's current location. But let's try your way and see if it works :)